templates/program/nph/admin/index.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'admin/base.html.twig' %}
+{% block title %}Admin - {% endblock %}
+{% block body %}
+    <ol class="breadcrumb">
+        <li><a href="{{ path('home') }}">Home</a></li>
+        <li class="active">Admin</li>
+    </ol>
+    <div class="page-header">
+        <h2><i class="fa fa-cog" aria-hidden="true"></i> Admin</h2>
+    </div>
+
+    <div class="row home-menu">
+        <div class="col-sm-6 col-md-4">
+            <a href="{# TODO: NPH Admin Sites #}" class="btn btn-lg btn-default btn-block"><i class="fa fa-hospital-o" aria-hidden="true"></i> Site Management</a>

```

Comment:
Could you link this to the admin site management page (nph_admin_sites) that I have created?